src/adapter/src/catalog/builtin.rs

```diff
@@ -1416,6 +1416,7 @@ pub static MZ_SINKS: Lazy<BuiltinTable> = Lazy::new(|| BuiltinTable {
         .with_column("type", ScalarType::String.nullable(false))
         .with_column("connection_id", ScalarType::String.nullable(true))
         .with_column("size", ScalarType::String.nullable(true))
+        .with_column("envelope_type", ScalarType::String.nullable(true))
```

Comment:
i feel like you could make it non-nullable, but this is technically more future-proof

Reply:
had the same thought! 😅